Added note about running javaagent in classpath (#8991)
DataDog/documentation: content/en/tracing/setup_overview/open_standards/java.md

}
```

> Aside from environment variables and system properties, there are additional configuration options as part of the `DDTracer.Builder` interface.  Consult the [Javadoc][2] for a full listing. Note the `dd-java-agent` should never be added to your classpath and can cause unexpected behavoir. 

Reviewer: ```suggestion
Aside from environment variables and system properties, there are additional configuration options as part of the `DDTracer.Builder` interface.  Consult the [Javadoc][2] for a full listing. 

**Note:** Never add `dd-java-agent` to your classpath. Doing so can cause unexpected behavior. 
```
